projects/common/element-components/slider.component.ts

```diff
@@ -0,0 +1,67 @@
+import { ElementComponent } from '../element-component.directive';
+import { SliderElement } from '../models/slider-element';
+import { FormElementComponent } from '../form-element-component.directive';
+import {ValidatorFn, Validators} from "@angular/forms";
+
+@Component({
+  selector: 'app-slider',
+  template: `
+    <div [style.display]="'flex'"
+         [style.flex-direction]="'row'"
+         [style.background-color]="elementModel.backgroundColor"
+         [style.width.%]="100"
+         [style.height.%]="100">
+      <div *ngIf="elementModel.showValues"
+           [style.color]="elementModel.fontColor"
+           [style.font-family]="elementModel.font"
+           [style.font-size.px]="elementModel.fontSize"
+           [style.line-height.%]="elementModel.lineHeight"
+           [style.font-weight]="elementModel.bold ? 'bold' : ''"
+           [style.font-style]="elementModel.italic ? 'italic' : ''"
+           [style.text-decoration]="elementModel.underline ? 'underline' : ''">{{elementModel.minValue | number:'.0'}}</div>
+      <div [style.display]="'flex'"
+           [style.flex-direction]="'column'"
+           [style.width.%]="100"
+           [style.height.%]="100">
+        <mat-slider
+          [formControl]="elementFormControl"
+          [disabled]="elementModel.readOnly"
+          [style.width.%]="100"
+          [max]="elementModel.maxValue"
+          [min]="elementModel.minValue">
+        </mat-slider>
+        <mat-error *ngIf="elementFormControl.touched && elementFormControl.errors">
+          {{elementFormControl.errors | errorTransform: elementModel}}
+        </mat-error>
+      </div>
+      <div *ngIf="elementModel.showValues"
+           [style.color]="elementModel.fontColor"
+           [style.font-family]="elementModel.font"
+           [style.font-size.px]="elementModel.fontSize"
+           [style.line-height.%]="elementModel.lineHeight"
+           [style.font-weight]="elementModel.bold ? 'bold' : ''"
+           [style.font-style]="elementModel.italic ? 'italic' : ''"
+           [style.text-decoration]="elementModel.underline ? 'underline' : ''"
+      >{{elementModel.maxValue | number:'.0'}}</div>
+    </div>
+  `,
+  styles: [
+    '.dynamic-image{width: 100%; height: fit-content}',
+    '.static-image{ width: 100%; height: 100%; object-fit: contain}',
+    ':host ::ng-deep .mat-slider-thumb{border-radius: 0; width: 10px; height: 40px; bottom: -15px}'
+  ]
+})
+export class SliderComponent extends FormElementComponent {
+  elementModel!: SliderElement;
+  // todo: ?? setting disabled attribute of slider may cause 'changed after checked' error
+  // todo: localize 'number' pipe to show '.' instead of ',' by values greater then 999
+
+  get validators(): ValidatorFn[] {
+    const validators: ValidatorFn[] = [];
+    if (this.elementModel.required) {
+      validators.push(Validators.min(this.elementModel.minValue + 1));
```

Comment:
Diese validierung über 'min' ist raffiniert, aber wirklich notwendig? Geht das nicht über 'required'? Können wir auch nochmal via Zoom drüber reden.
Die Unterscheidung in der Pipe ist sowieso fragwürdig und kommt (hoffentlich) weg.

Reply:
Hm wir werden noch andere Fälle für die Pipe haben, wo required übersetzt werden muss - finde ich nicht schlimm. Wie willst Du sonst anders required für den Slider interpretieren, als dass mehr der Minimalwert gesetzt ist? Das ist so ähnlich wie die Checkbox, wo Du auch "nicht gesetzt" als invalid markierst, obwohl es natürlich ein valider Wert ist. Das Problem mit touched hatte ich deshalb auch.

Reply:
Das müssen wir mal mit Achim diskutiern, ich stecke da nicht so drin. Ich habe nur vorhin die Fallunterscheidung dadrin gesehen und bemängelt. Achim sieht das ähnlich und hat ein Refactoring vorgenommen:

https://github.com/iqb-berlin/verona-modules-aspect/commit/75b71f2525d1d9a7dd6aa4d2c1e77f6b16c172e9

Wie wir das hier bei dir anpassen, müssen wir mal gemeinsam schauen.

Reply:
Im Prinzip wird die Pipe nur gebraucht wenn es mehrere mögliche Warnmeldungen gibt, um diese hintereinander darzustellen Das trifft für den Slider nicht zu. Deshalb wird sie dort gar nicht gebraucht und braucht auch keine Fallunterscheidung. 
<mat-error *ngIf="elementFormControl.errors && elementFormControl.touched"
                 class="error-message">
        {{elementModel.requiredWarnMessage}}
      </mat-error>
sollte reichen